get-started/index.md

```diff
@@ -90,29 +98,29 @@ You can create the resource graphically in the OperatorHub UI after following [I
   image-name="cryostat-resources-after.png"
 %}
 
-You can also create the resource manually using a YAML definition like the following:
+You can also create the `CR` manually using a YAML definition like the following:
 
 ```yaml
```

Comment:
Can we use this: https://github.com/cryostatio/cryostat-operator/blob/main/config/samples/operator_v1beta1_cryostat.yaml

This should be the default YAML when creating Cryostat.

![Screenshot from 2023-05-13 23-52-52](https://github.com/cryostatio/cryostatio.github.io/assets/68053619/cf4baebb-a2b5-449d-8d98-6f6e7fb798b5)